lib/routers/profile/pil.js

```diff
@@ -8,7 +8,7 @@ const router = Router({ mergeParams: true });
 const submit = (action) => {
   return (req, res, next) => {
     const params = {
-      action,
+      action: action || req.params.action,
```

Comment:
hmmm, this means any action will be passed to workflow, what if the action is `delete`?

Reply:
I was supposed to be applying a check at some point - good catch

Reply:
added a `validateAction` for now - we probably want to tie this to user perms somehow but I need to discuss with you lot.